### ISSUE TITLE
BLR post process code

### ISSUE THREAD

**Issue by breezelj** [Scalsol/mega.pytorch#16]
thanks for your work. when reading the test_net.py code, I do not find any code about BLR post process. Hasn't this part been released yet。 or just I miss somewhere and could you point out where this BLR code is. thanks your reply.

**Comment by Scalsol** [comment]
Yes, the BLR post process code is not available at this moment. I'll add it after NIPS deadline.

**Comment by breezelj** [comment]
thanks, I am looking forward to it.

**Comment by PaPaQQQ** [comment]
I still haven‘t found any code about BLR post process. could add it?
